### ISSUE TITLE
The image will not go up.

### ISSUE THREAD

**Issue by tomzangsad** [GeyserMC/Geyser#3222]
### Describe the bug

The image will not go up.

### Screenshots / Videos

![image](https://user-images.githubusercontent.com/63664294/183973206-00b5768d-b636-4bb3-b30b-601fd0aecb26.png)

### Server Version and Plugins

git-Purpur-1752 (MC: 1.19.2)

### Geyser Version

 [feature/extensions] 2.1.0-SNAPSHOT (git-local/dev-b9646b9) 

### Minecraft: Bedrock Edition Device/Version

1.19.20

**Comment by Camotoy** [comment]
Will likely be fixed once master is merged into extensions.

**Comment by tomzangsad** [comment]
When will it be merged? can tell me

**Comment by Camotoy** [comment]
Probably by tonight - there's one last fix I want to get in.

**Comment by Camotoy** [comment]
Should be fixed.
